src/core/models/outputs/output_extended.c

```diff
@@ -11,36 +11,59 @@
 #include "utlist.h"
 
 // minimum dust allowance
-#define MIN_DUST_ALLOWANCE 1000000
+// #define MIN_DUST_ALLOWANCE 1000000
 
 // maximum number of feature blocks
 #define MAX_FEATURE_BLOCKS_COUNT 8
```

Comment:
Should `MAX_FEATURE_BLOCKS_COUNT` be removed because it's redundant? `MAX_EXTENDED_FEATURE_BLOCKS_COUNT` is new one.

Reply:
yes :+1: